craft_parts/parts.py

```diff
@@ -1073,6 +1097,20 @@ def part_dependencies(
     return dependencies
 
 
+def part_dependencies_overlay_bootstrap(
+    part: Part, *, part_list: list[Part]
+) -> set[Part]:
+    """Return a set of all the parts upon which the named part depends and bootstrapping the overlay.
```

Comment:
This is a bit confusing to me, do you mean "and _that are_ bootstrapping the overlay"?